### PR TITLE
MGMT-1722 Modify 2nd ignition to create parameter file to Openshift for DHCP automatic VIP allocation

### PR DESCRIPTION
**Pull request by ori-amizur**
/cc @eranco74 
/cc @ronniel1 
/cc @yevgeny-shnaidman 
/cc @YuviGold

### CODE REVIEW

**Comment by romfreiman** [comment]
@ori-amizur can u please open a ticket to change this funcitonality and link it to this epic: https://issues.redhat.com/browse/MGMT-1780

File: render_files.py [hunk 1]
```diff
@@ -175,6 +191,7 @@ def main():
     install_config = os.environ.get("INSTALLER_CONFIG")
     cluster_id = os.environ.get("CLUSTER_ID")
     inventory_endpoint = os.environ.get("INVENTORY_ENDPOINT")
+    dhcp_allocation_file = os.environ.get("DHCP_ALLOCATION_FILE")
```
Comment (romfreiman) [inline render_files.py:194]: how does it related to BMH? and I would call DHCP_ALLOCATION_FILE diffrently @ori-amizur

Comment (ori-amizur) [reply render_files.py:194]: It is not.  Why do you think it is?

Comment (romfreiman) [reply render_files.py:194]: because it's in the same function?

**Pull request merged**